lib/puppet/parser/functions/strftime.rb

```diff
@@ -135,8 +135,14 @@
 # Notice the timestamp using a format that notices the ISO 8601 date format
 notice($timestamp.strftime('%F')) # outputs '2016-08-24'
 
-# Notice the timestamp using a format that notices weekday, month, day, time, and year
-notice($$timestamp.strftime('%c')) # outputs 'Wed Aug 24 12:13:14 2016'
+# Notice the timestamp using a format that notices weekday, month, day, time (as UTC), and year
+notice($timestamp.strftime('%c')) # outputs 'Wed Aug 24 12:13:14 2016'
+
+# Notice the timestamp using a specific timezone
+notice($timestamp.strftime('%F %T %z', 'PST')) # outputs '2016-08-24 04:13:14 -0800'
+
+# Notice the timestamp using timezone that is current for the evaluating process
+notice($timestamp.strftime('%F %T', 'CURRENT')) # outputs the timestamp using the timezone for the current process
```

Comment:
This has 'CURRENT' in upper case. Elsewhere the same was stated in lower case 'current' - which is it (specific case, or case independent)?